apps/prairielearn/src/tests/sprocs/check_course_instance_access.test.sql

```diff
@@ -142,6 +142,6 @@ FROM
     user_institution_id,
     course_institution_id,
     $date
-  ) AS authorized
+  ) AS authorized -- noqa: AL05
```

Comment:
This was a false positive with https://docs.sqlfluff.com/en/stable/reference/rules.html#rule-AL05, since this isn't a table but a column.